### PR TITLE
Add missing "using" statement to example in the readme close #1044

### LINKED ISSUE #1044: Example in the readme might be missing an "using" statement
### Steps to reproduce

try to setup pomelo as specified by the example in the readme

### The issue

The project doesn't build because ServerVersion is not in scope.

```
Microsoft (R) Build Engine version 16.4.0+e901037fe for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restore completed in 23.96 ms for /home/abraham/src/OpenPaymentsService/HubDePagos.csproj.
Installers/DbInstaller.cs(19,68): error CS0246: The type or namespace name 'ServerVersion' could not be found (are you missing a using directive or an assembly reference?) [/home/abraham/src/OpenPaymentsService/HubDePagos.csproj]
Startup.cs(54,13): warning MVC1005: Using 'UseMvc' to configure MVC is not supported while using Endpoint Routing. To continue using 'UseMvc', please set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices'. [/home/abraham/src/OpenPaymentsService/HubDePagos.csproj]

Build FAILED.

Startup.cs(54,13): warning MVC1005: Using 'UseMvc' to configure MVC is not supported while using Endpoint Routing. To continue using 'UseMvc', please set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices'. [/home/abraham/src/OpenPaymentsService/HubDePagos.csproj]
Installers/DbInstaller.cs(19,68): error CS0246: The type or namespace name 'ServerVersion' could not be found (are you missing a using directive or an assembly reference?) [/home/abraham/src/OpenPaymentsService/HubDePagos.csproj]
    1 Warning(s)
    1 Error(s)

Time Elapsed 00:00:01.67
```

### Further technical details

MySQL version: Mariadb 10.14.12
Operating system: Archlinux
Pomelo.EntityFrameworkCore.MySql version: 3.0.1
Microsoft.AspNetCore.App version: 3.1.2

### hypothesis

an statement like this is missing:

`using Pomelo.EntityFrameworkCore.MySql.Storage;`

at least that worked for me

### PR DESCRIPTION
**Pull request by categulario**
I tried to use it as it was and it failed. Maybe this can save time to someone in the future

### CODE REVIEW

**Comment by lauxjpn** [comment]
Thinking about it, it is probably cleaner to just use another overload:

```c#
services.AddDbContextPool<YourDbContext>(options => options
    // replace with your connection string
    .UseMySql("Server=localhost;Database=ef;User=root;Password=1234;",
              mySqlOptions => mySqlOptions
                  // replace with your Server Version and Type
                  .ServerVersion(new Version(8, 0, 20), ServerType.MySql)
    )
);
```

We should consolidate some of the namespaces for .NET 5.

File: README.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ Add `Pomelo.EntityFrameworkCore.MySql` to the services configuration in your the
 using System;
 using Microsoft.EntityFrameworkCore;
 using Pomelo.EntityFrameworkCore.MySql.Infrastructure;
+using Pomelo.EntityFrameworkCore.MySql.Storage;
```
Comment (lauxjpn) [inline README.md]: Thinking about it, it is probably cleaner to just use another overload (which we did previously) in the sample code:

```c#
services.AddDbContextPool<YourDbContext>(options => options
    // replace with your connection string
    .UseMySql("Server=localhost;Database=ef;User=root;Password=1234;",
              mySqlOptions => mySqlOptions
                  // replace with your Server Version and Type
                  .ServerVersion(new Version(8, 0, 20), ServerType.MySql)
    )
);
```

We should/will consolidate some of the namespaces for .NET 5 and this way, people will not get compile time errors for this one.

Comment (categulario) [reply README.md]: Whatever works. I just didn't want another developer to spend 15 minutes trying to figure out why the code in the example (being so simple) didn't compile.

Comment (lauxjpn) [reply README.md]: If you change it, I merge it.

Comment (categulario) [reply README.md]: changed

**Pull request merged**